### ISSUE TITLE
Babel error in next.6

### ISSUE THREAD

**Issue by ForsakenHarmony** [developit/preact-cli#605]
**What is the current behaviour?**

`Error: [BABEL] .../project/preact.config.js: .babelrc is only allowed in root programmatic options, or babel.config.js/config file options`

**If the current behaviour is a bug, please provide the steps to reproduce.**

Install `preact-cli@next`
add a `.babelrc` with `{ "presets": ["preact-cli/babel"] }`

**What is the expected behaviour?**

No errors

**Please mention other relevant information.**
- CLI version: `v3.0.0-next.6`

Question is: what's the proper way to fix this?

Deleting this line seems to fix it, but is it the proper way to do it with babel 7?
https://github.com/developit/preact-cli/blob/master/src/lib/babel-config.js#L5

**Comment by prateekbh** [comment]
does `--esm` works after deleting that line?

**Comment by ForsakenHarmony** [comment]
also 
```
$ preact build --esm --no-prerender
Adding @babel/preset-env because it was not found
Error: Promise rejected with value: '\u001b[31mBuild failed! null\u001b[39m'
    at project\node_modules\loud-rejection\index.js:27:12
    at Array.forEach (<anonymous>)
    at EventEmitter.<anonymous> (project\node_modules\loud-rejection\index.js:23:24)
    at EventEmitter.emit (events.js:182:13)
    at emit (project\node_modules\signal-exit\index.js:77:11)
    at processEmit [as emit] (project\node_modules\signal-exit\index.js:150:5)
```

**Comment by prateekbh** [comment]
ohhh!!!! i'll check

**Comment by ForsakenHarmony** [comment]
it's coming from the esm plugin

**Comment by prateekbh** [comment]
is this still happening?

**Comment by ForsakenHarmony** [comment]
Do you have a `.babelrc`?

**Comment by oieduardorabelo** [comment]
@ForsakenHarmony , seems this issue is fixed on `"preact-cli": "3.0.0-next.9"`

my steps:

- `$ preact create simple testy`
- `$ cd testy`
- `$ npm install preact-cli@3.0.0-next.9`
- `$ echo "{ \"presets\": [\"preact-cli/babel\"] }" > .babelrc`
- `$ npm run start`

#### screens

<img width="390" alt="screen shot 2018-09-12 at 6 24 36 pm" src="https://user-images.githubusercontent.com/829902/45406051-4bb29580-b6b9-11e8-8b3f-e41e8331d9d5.png">

![00](https://user-images.githubusercontent.com/829902/45406049-4b19ff00-b6b9-11e8-8080-897017771e71.png)
![01](https://user-images.githubusercontent.com/829902/45406050-4bb29580-b6b9-11e8-86eb-21b81ae8078d.png)

**Comment by ForsakenHarmony** [comment]
building also works? in that case I could close this
